cloud-providers/azure/template/scripts/setup_payara.sh

```diff
@@ -16,7 +56,7 @@ function install_packages
 function download_payara
 {
     cd $HOME_DIR
-    rm -rf payara5
+    rm -rf payara6
     wget http://central.maven.org/maven2/fish/payara/distributions/payara/5.192/payara-5.192.zip
```

Comment:
This probably won't work. For P6, it's https://repo.maven.apache.org/maven2/fish/payara/distributions/payara/6.2022.1/payara-6.2022.1.zip